crates/holochain/src/sweettest/sweet_topos/edge.rs

```diff
@@ -0,0 +1,98 @@
+use super::network::NetworkTopologyConductor;
+use super::node::NetworkTopologyNode;
+use crate::sweettest::SweetConductor;
+use arbitrary::Arbitrary;
+use holochain_zome_types::prelude::CellId;
+use rand::Rng;
+use std::hash::{Hash, Hasher};
+
+/// A network edge in a network topology. Represents a network connection.
+/// Edges are directed, so if you want a bidirectional connection you need two
+/// edges.
+#[derive(Arbitrary, Clone, Debug, Default, Eq)]
+pub struct NetworkTopologyEdge {
+    id: [u8; 32],
+    source_conductor: NetworkTopologyConductor,
+    target_conductor: NetworkTopologyConductor,
+    cells: Vec<CellId>,
```

Comment:
@jost-s because that's how agent info works, edges represent the view from one conductor on another, there's nothing that guarantees that some conductor has a perfect view on another conductor, in the general case we want to be able to represent subsets and supersets of what is real

Reply:
I see. I'm asking because we're using this topology in tests, and in tests I'll know when mutations are going to happen, so I could store the cells of the target conductor in a vector before the mutation to have them available after the mutation. I haven't seen a test here where you'd need the cells. I'm not opposing this field, just want to understand the design choice.